riot_api_helpers.py

```diff
@@ -42,4 +42,66 @@ def get_summoner_by_name(summoner_name):
             "summoner_level": json_response_content["summonerLevel"]
         }
     except (KeyError, TypeError, ValueError):
-        return None
\ No newline at end of file
+        return None
+
+def get_matches_for_account(account_id, champion=None, queue=None, season=None, endTime=None, beginTime=None, endIndex=None, beginIndex=None):
+    """Requests a MatchlistDto from the Riot Match API. This contains a series of MatchReferenceDtos, each containing:
+    gameId: long
+    role: string
+    season: int
+    platformId: string
+    champion: int
+    queue: int
+    lane: string
+    timestamp: long
+    
+    Args:
+        account_id (string): Encrypted accountId. This is returned in the summonerDto from the get_summoner_by_name function.
+        champion (int, optional): Champion key. The value that corresponds to each champion can be found in champions.json. Defaults to None.
+        queue (int, optional): Queue key. The value that corresponds to each queue can be found in queues.json.  Defaults to None.
+        season (int, optional): Corresponds to the season. Riot's API documentation specifies this as deprecated, so do not rely on it. Defaults to None.
+        endTime (long, optional): The end time to use for filtering the matchlist specified as epoch milliseconds. If beginTime is specified but not endTime, endTime defaults to the current timestamp in milliseconds. Defaults to None.
+        beginTime (long, optional): The begin time to use for filtering the matchlist specified as epoch milliseconds. If beginTime is specified but not endTime, endTime defaults to the current timestamp in milliseconds. Defaults to None.
+        endIndex (int, optional): The end index to use for filtering the matchlist. If beginIndex is specified but not endIndex, endIndex defaults to beginIndex+100. The maximum range allowed is 100. Defaults to None.
+        beginIndex (int, optional): The begin index to use for filtering the matchlist. If beginIndex is specified but not endIndex, endIndex defaults to beginIndex+100. The maximum range allowed is 100. Defaults to None.
+
+    Returns:
+        [dict]: Returns a dictionary containing a list of match references, each denoting a game played. This is followed by values for the keys: startIndex, endIndex, totalGames.
+    """
+    api_key = os.getenv("API_KEY")
```

Comment:
This can be moved out into the main file rather than assigning in each function.